### ISSUE TITLE
Add support for ID(node) in queries

### ISSUE THREAD

**Issue by christyharagan** [swilly22/redis-graph#34]
Currently it is not possible to get the ID of a node returned from a query. So it is currently not possible to run queries such as:

```
GRAPH.QUERY mygraph "MATCH (src:A)-[E]->(tgt:B) RETURN ID(src)"
```

**Comment by swilly22** [comment]
Agree, I understand the need for getting a node's ID,
Added to features list.

**Comment by swilly22** [comment]
Will be implemented as part of #70

**Comment by swilly22** [comment]
Done as part #78
